save-frontend/src/main/kotlin/com/saveourtool/save/frontend/utils/RequestUtils.kt

```diff
@@ -51,6 +51,11 @@ interface WithRequestStatusContext {
     fun setLoadingCounter(transform: (oldValue: Int) -> Int)
 }
 
+/**
+ * @return message part of json response
```

Comment:
Please mark that this is applicable for Spring Boot's error objects

Reply:
sure

Reply:
Done